### PR TITLE
chore(macros/HTML): update zh-CN strings

### PR DESCRIPTION
**Pull request by awxiaoxian2020**

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
cc: @yin1999

**Comment by yin1999** [comment]
I'll review it later today

File: kumascript/macros/HTMLSidebar.ejs [hunk 1]
```diff
@@ -159,7 +159,44 @@ var text = mdn.localStringMap({
...
     'The_MDN_project': 'Проект MDN',
-  }
+  },
+  'zh-CN': {
+    'Tutorials': '教程',
+    'HTML_basics': 'HTML 基础',
+    'Introduction_to_HTML': 'HTML 介绍',
+      'Introduction_to_HTML_overview': 'HTML 概述',
+      'Getting_started_with_HTML': '开始学习 HTML',
+      'Whats_in_the_head_metadata_in_HTML': 'head 里面是什么？HTML 中的元数据',
```
Comment (yin1999) [inline kumascript/macros/HTMLSidebar.ejs:169]:
```suggestion
      'Whats_in_the_head_metadata_in_HTML': 'head 标签里面有什么？HTML 中的元数据',
```

File: kumascript/macros/HTMLSidebar.ejs [hunk 2]
```diff
@@ -159,7 +159,44 @@ var text = mdn.localStringMap({
...
+      'HTML_text_fundamentals': 'HTML 文本基础',
+      'Creating_hyperlinks': '建立超链接',
+      'Advanced_text_formatting': '高级文本格式',
+      'Document_and_website_structure': '文档和网站架构',
+      'Debugging_HTML': 'HTML 调试',
+      'Assessment_Marking_up_a_letter': '测验：标记信件',
+      'Assessment_Structuring_a_page_of_content': '测验：构建出有内容的网页',
```
Comment (yin1999) [inline kumascript/macros/HTMLSidebar.ejs:176]:
```suggestion
      'Assessment_Structuring_a_page_of_content': '测验：构建网页内容',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 3]
```diff
@@ -1065,28 +1065,28 @@ var text = mdn.localStringMap({
     'HTML_Structuring_the_web': 'HTML — 构建 Web',
       'Introduction_to_HTML': 'HTML 介绍',
         'Introduction_to_HTML_overview': 'HTML 概述',
-        'Getting_started_with_HTML': 'HTML 入门',
-        'Whats_in_the_head_metadata_in_HTML': '脑袋里是什么？HTML 中的元数据',
-        'HTML_text_fundamentals': 'HTML 文字基础',
+        'Getting_started_with_HTML': '开始学习 HTML',
+        'Whats_in_the_head_metadata_in_HTML': 'head 里面是什么？HTML 中的元数据',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1069]:
```suggestion
        'Whats_in_the_head_metadata_in_HTML': 'head 标签里面有什么？HTML 中的元数据',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 4]
```diff
@@ -1065,28 +1065,28 @@ var text = mdn.localStringMap({
...
-        'Advanced_text_formatting': '高级文字格式',
-        'Document_and_website_structure': '文档与网站架构',
-        'Debugging_HTML': 'HTML 除错',
-        'Assessment_Marking_up_a_letter': '作业：标记字母',
-        'Assessment_Structuring_a_page_of_content': '作业：构建出有内容的网页',
+        'Advanced_text_formatting': '高级文本格式',
+        'Document_and_website_structure': '文档和网站架构',
+        'Debugging_HTML': 'HTML 调试',
+        'Assessment_Marking_up_a_letter': '测验：标记信件',
+        'Assessment_Structuring_a_page_of_content': '测验：构建出有内容的网页',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1076]:
```suggestion
        'Assessment_Structuring_a_page_of_content': '测验：构建网页内容',
```

File: kumascript/macros/HTMLSidebar.ejs [hunk 5]
```diff
@@ -159,7 +159,44 @@ var text = mdn.localStringMap({
...
+    'Multimedia_and_embedding': '多媒体与嵌入',
+      'Multimedia_and_embedding_overview': '多媒体与嵌入概述',
+      'Images_in_HTML': 'HTML 中的图片',
+      'Video_and_audio_content': '视频和音频内容',
+      'From_object_to_iframe_other_embedding_technologies': '从对象到 iframe——其他嵌入技术',
+      'Adding_vector_graphics_to_the_web': '为 Web 新增矢量图',
+      'Responsive_images': '自适应图片',
```
Comment (yin1999) [inline kumascript/macros/HTMLSidebar.ejs:183]:
```suggestion
      'Responsive_images': '响应式图片',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 6]
```diff
@@ -1065,28 +1065,28 @@ var text = mdn.localStringMap({
...
       'Multimedia_and_embedding': '多媒体与嵌入',
         'Multimedia_and_embedding_overview': '多媒体与嵌入概述',
         'Images_in_HTML': 'HTML 中的图片',
         'Video_and_audio_content': '视频和音频内容',
-        'From_object_to_iframe_other_embedding_technologies': '从对象到 iframe — 其他嵌入技术',
-        'Adding_vector_graphics_to_the_web': '为 Web 新增向量图',
+        'From_object_to_iframe_other_embedding_technologies': '从对象到 iframe——其他嵌入技术',
+        'Adding_vector_graphics_to_the_web': '为 Web 新增矢量图',
         'Responsive_images': '自适应图片',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1083]:
```suggestion
        'Responsive_images': '响应式图片',
```

File: kumascript/macros/HTMLSidebar.ejs [hunk 7]
```diff
@@ -159,7 +159,44 @@ var text = mdn.localStringMap({
...
+      'HTML_table_basics' : 'HTML 表格基础',
+      'HTML_table_advanced_features_and_accessibility' : 'HTML 高级表格特性和无障碍',
+      'Assessment_Structuring_planet_data' : '测验：构建行星数据',
+    'Reference': '参考：',
+    'HTML_Elements': 'HTML 元素',
+    'Global_attributes': '全局属性',
+    'Input_types': '<code>&lt;input&gt;</code> types',
+    'Documentation': '文档：',
+    'Useful_lists': '附表',
+    'Index': 'HTML 页面索引',
```
Comment (yin1999) [inline kumascript/macros/HTMLSidebar.ejs:196]:
```suggestion
    'Index': 'HTML 文档索引',
```

**Comment by yin1999** [comment]
some suggestions

**Comment by awxiaoxian2020** [comment]
> some suggestions

We should update the title in some time.

**Comment by yin1999** [comment]
LGTM. It's time to poke @mdn/mdn-community-engagement

**Review by schalkneethling — APPROVED** [review]
Thank you @awxiaoxian2020 and @yin1999 🎉

**Comment by schalkneethling** [comment]
@awxiaoxian2020 for future reference, this repository requires signed commits. You can read more about it here: https://docs.github.com/articles/about-gpg/

**Comment by awxiaoxian2020** [comment]
> @awxiaoxian2020 for future reference, this repository requires signed commits. You can read more about it here: https://docs.github.com/articles/about-gpg/

Thanks for your reminding! I add the GPG key now!

**Pull request merged**